### PR TITLE
View an individual list

### PR DESCRIPTION
**Pull request by andyw8**
Some of this markup is similar to that on the lists page, but I think
it's a little premature to extract a new partial.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -27,5 +27,7 @@ en:
   lists:
     list:
       empty: This list is empty
+    show:
+      empty: This list is empty
```
Comment (andyw8) [inline config/locales/en.yml]: Even thought this is the same message as for the lists page, I would treat this as coincidental duplication. I think there are many reasons why the text in each place should differ in future.

File: config/locales/en.yml [hunk 2]
```diff
@@ -27,5 +27,7 @@ en:
   lists:
     list:
       empty: This list is empty
```
Comment (taylorrf) [inline config/locales/en.yml]: I think we can change this message to "We don't have any curated list yet." and keep the message of show action.

**Comment by taylorrf** [comment]
I think you can rebase this branch with the current master to solve the conflict what Github wants to solve.

**Comment by taylorrf** [comment]
Code looks good, take a look on my two comments and I think we'll be ready to merge it.

File: config/locales/en.yml (hunk 2 above)
Comment (andyw8) [inline config/locales/en.yml]: Updated as per your suggestion.

**Pull request merged**